src/layouts/ResourceCategory/components/ResourceCard.tsx

```diff
@@ -30,6 +31,17 @@ interface ResourceCardProps {
   resourceType: ResourcePageData["resourceType"]
 }
 
+const getIcon = (resourceType: string): JSX.Element => {
+  switch (resourceType) {
+    case "post":
+      return <Icon as={BiFileBlank} fontSize="1.5rem" fill="icon.alt" />
+    case "link":
+      return <Icon as={BiLink} fontSize="1.5rem" fill="icon.alt" />
+    default:
```

Comment:
nit: could we change `default:` to `case "file":` for clarity?

Reply:
Changed in https://github.com/isomerpages/isomercms-frontend/pull/1064/commits/36a32368a50a92cf4a0da92b1b2f795d49f0a0aa!